### PR TITLE
T15812 - drop XSLT rules

### PR DESCRIPTION
**Pull request by krnowak**
https://phabricator.endlessm.com/T15812

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -156,25 +156,20 @@ eos_updater_SOURCES = \
 
 dist_man8_MANS += docs/eos-updater.8
 
+eos_updater_dbus_if := com.endlessm.Updater
```
Comment (pwithnall) [inline src/Makefile.am]: Probably doesn’t make sense to have this as a variable. Changing the interface name would be a major change to all of the codebase, so factoring it out adds no value, and makes things less greppable (for example, `git grep com.endlessm.Updater.service` doesn’t work).

File: src/Makefile.am [hunk 2]
```diff
@@ -156,25 +156,20 @@ eos_updater_SOURCES = \
...
+dbussystemservice_DATA = $(srcdir)/$(eos_updater_dbus_if).service
 
-CLEANFILES = $(BUILT_SOURCES) $(dbusconf_DATA) $(dbussystemservice_DATA)
+CLEANFILES = $(BUILT_SOURCES)
 
 EXTRA_DIST = \
-	eos-updater-ifname.xslt \
-	eos-updater-policy.xslt \
-	eos-updater-service.xslt \
 	eos-updater.xml \
```
Comment (pwithnall) [inline src/Makefile.am]: It would be useful to also install `eos-updater.xml` as `/usr/share/dbus-1/interfaces/com.endlessm.Updater.xml`. Please do that as a separate commit.

File: src/Makefile.am [hunk 3]
```diff
@@ -156,25 +156,20 @@ eos_updater_SOURCES = \
...
 
+eos_updater_dbus_if := com.endlessm.Updater
+
 dbusconfdir = $(datadir)/dbus-1/system.d
 dbussystemservicedir = $(datadir)/dbus-1/system-services
 
-dbusconf_DATA = $(eos_updater_dbus_if).conf
-dbussystemservice_DATA = $(eos_updater_dbus_if).service
+dbusconf_DATA = $(srcdir)/$(eos_updater_dbus_if).conf
+dbussystemservice_DATA = $(srcdir)/$(eos_updater_dbus_if).service
```
Comment (pwithnall) [inline src/Makefile.am]: Shouldn’t `dbusconf_DATA` and `dbussystemservice_DATA` both have a `dist_` prefix? Please double-check that `make distcheck` and building the Debian package works after applying these commits. (As I said on Slack, I use [`build-snapshot`](https://git.apertis.org/cgit/apertis-customizations.git/tree/development/build-snapshot) to do both.)

File: debian/control [hunk 4]
```diff
@@ -23,7 +23,6 @@ Build-Depends:
  libsoup2.4-dev,
  libsystemd-dev,
  ostree,
- xsltproc,
```
Comment (pwithnall) [inline debian/control]: This commit can be squashed into the main one. No need to commit to the `debian/` directory separately.

Comment (krnowak) [inline debian/control]: I was doing separate commits for changes in the debian directory for the sake of "debian: " prefix in the commit message. :)

File: src/Makefile.am (hunk 1 above)
Comment (krnowak) [inline src/Makefile.am]: Will do.

File: src/Makefile.am (hunk 3 above)
Comment (krnowak) [inline src/Makefile.am]: Hm hm hm, `make distcheck` passed here, I did run it before filing this PR. I think these things are distributed automatically if they are not built and installed somewhere, but that sounds overly smart/magic, so I'll check it.

File: src/Makefile.am (hunk 2 above)
Comment (krnowak) [inline src/Makefile.am]: Alright.

**Comment by krnowak** [comment]
Updated.

**Comment by pwithnall** [comment]
Lovely, thanks.
